coops-ui/src/resources/config-objects.ts

```diff
@@ -0,0 +1,68 @@
+
+export class Business {
+     typeEnum: string;
+     displayName:String;
```

Comment:
just need a space here after the colon, and a lowercase s in `string`.

Reply:
Yes to all.

FYI, use `string` as a type to declare properties or `String` when you want to use the class prototype methods directly (eg, new String() or String.xxx).